src/app/modules/group/components/group-leave/group-leave.component.html

```diff
@@ -0,0 +1,18 @@
+<p i18n>
+  <ng-container *ngIf="locked; else allow">
+    You are currently not allowed to leave this group.
+  </ng-container>
+  <ng-template #allow>
+    You can freely leave this group. Note that you may lose access to content if this group was granting some permissions to its members.
+  </ng-template>
+</p>
```

Comment:
I think i18n attribute should be at `ng-container` and `ng-template` level, otherwise it creates translations like [this](https://github.com/France-ioi/AlgoreaFrontend/pull/863/files#diff-03f42e151f2e4a3a58f3c395b81018f817bc913a5d68a6006e09cd293564c864R1617)
```xml
<source>
  <x id="START_TAG_NG_CONTAINER" ctype="x-ng_container" equiv-text="&lt;ng-container *ngIf=&quot;locked; else allow&quot;>"/>
    You are currently not allowed to leave this group. 
  <x id="CLOSE_TAG_NG_CONTAINER" ctype="x-ng_container" equiv-text="&lt;/ng-container>"/>
  <x id="START_TAG_NG_TEMPLATE" ctype="x-ng_template" equiv-text="&lt;ng-template #allow>"/>
    You can freely leave this group. Note that you may lose access to content if this group was granting some permissions to its members. 
  <x id="CLOSE_TAG_NG_TEMPLATE" ctype="x-ng_template" equiv-text="&lt;/ng-template>"/>
</source>
<target state="new">
  <x id="START_TAG_NG_CONTAINER" ctype="x-ng_container" equiv-text="&lt;ng-container *ngIf=&quot;locked; else allow&quot;>"/>
    You are currently not allowed to leave this group. 
  <x id="CLOSE_TAG_NG_CONTAINER" ctype="x-ng_container" equiv-text="&lt;/ng-container>"/>
  <x id="START_TAG_NG_TEMPLATE" ctype="x-ng_template" equiv-text="&lt;ng-template #allow>"/>
    You can freely leave this group. Note that you may lose access to content if this group was granting some permissions to its members.
  <x id="CLOSE_TAG_NG_TEMPLATE" ctype="x-ng_template" equiv-text="&lt;/ng-template>"/>
</target>
```
instead of just
```xml
<source>  You are currently not allowed to leave this group. </source>
<target state="new"> You are currently not allowed to leave this group. </target>
```
and
```xml
<source>
  You can freely leave this group. Note that you may lose access to content if this group was granting some permissions to its members.
</source>
<target state="new">
  You can freely leave this group. Note that you may lose access to content if this group was granting some permissions to its members.
</target>
```

Reply:
Fixed